docs/cuproj/source/user_guide/cuproj_api_examples.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c5fdf490-fa77-4e56-92d1-53101fff75ba",
+   "metadata": {},
+   "source": [
+    "# cuProj Python User's Guide\n",
+    "\n",
+    "cuProj is a GPU-accelerated Python library for cartographic coordinate projection and coordinate transformations between coordinate reference systems (CRS). The cuProj Python API provides an accessible interface to high-performance projections accelerated by CUDA-enabled GPUs. The API closely follows the [PyProj](https://pyproj4.github.io/pyproj/stable/) API."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "caadf3ca-be3c-4523-877c-4c35dd25093a",
+   "metadata": {},
+   "source": [
+    "## Contents\n",
+    "\n",
+    "This guide provides a working example for all of the python API components of cuProj.  \n",
+    "The following list links to each subsection.\n",
+    "\n",
+    "* [Installing cuProj](#Installing-cuProj)\n",
+    "* [Transformations with Transformer](#Transformations-with-Transformer)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "115c8382-f83f-476f-9a26-a64a45b3a8da",
+   "metadata": {},
+   "source": [
+    "## Installing cuProj\n",
+    "Read the [RAPIDS Quickstart Guide](  https://rapids.ai/start.html     ) to learn more about installing all RAPIDS libraries, including cuProj.\n",
```

Comment:
```suggestion
    "Read the [RAPIDS Quickstart Guide](https://docs.rapids.ai/install) to learn more about installing all RAPIDS libraries, including cuProj.\n",
```
Updating to direct link since that's where it redirects to.

Also, maybe not the spot for this, but should we be adding cuProj to the rapids metapackages? I'm somewhat assuming its too late in `23.08` but maybe in `23.10` we get that in.

I'll also need to add cuProj to conda/pip options in the selector in https://github.com/rapidsai/docs/pull/414, right?

Reply:
Yes please